Aquarius/Src/Aquarius/Core/Input.h

```diff
@@ -0,0 +1,168 @@
+#include <cstdint>
+
+#include "Window.h"
+
+namespace Aquarius {
+	enum class KeyState : int
+	{
+		Released = 0,
+		Pressed = 1
+	};
+
+	using MouseButtonState = KeyState;
+
+	enum class KeyCode : int
```

Comment:
Looks good, once you merge I will update my refinement branch so I can use these in the event files.